.github/CODEOWNERS

```diff
@@ -114,6 +114,7 @@ clang/test/AST/Interp/ @tbaederr
 # MLIR NVVM Dialect in MLIR
 /mlir/**/LLVMIR/**/BasicPtxBuilderInterface* @grypp
 /mlir/**/NVVM*/ @grypp
+/mlir/**/NVVM* @grypp
```

Comment:
Is this making the one on the line before obsolete?

Reply:
I think the first one catches /mlir/**/Target/LLVM/NVVM/Target.* files, but I haven't implemented them, so I have removed that line